vnet/spec/vnet/endpoints/1.0/network_services_spec.rb

```diff
@@ -35,12 +35,10 @@ def app
 
   describe "PUT /:uuid" do
     let!(:new_interface) { Fabricate(:interface) { uuid "if-other"}  }
+
     accepted_params = {
-      :interface_uuid => "if-other",
-      :incoming_port => 40,
-      :outgoing_port => 100
```

Comment:
Here too these parameters are still accepted in the web api so they shouldn't be removed from the test.